t/admin/credentials.t

```diff
@@ -110,7 +110,7 @@ passed
                      "desc": "basic-auth for jack",
                      "plugins": {
                          "basic-auth": {
-                             "username": "the-user",
+                             "username": "the-new-user",
```

Comment:
After adding the duplicate judgment, it cannot be added here again, so I made a modification.